config/prod/ecmonsen-code-storage-s3.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: ecmonsen-code
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "eva.monsen@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) Synapse username (default ""), required if AllowWriteBucket=true
+  #  SynapseUserName: 'jsmith'
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  #  GrantAccess:
```

Comment:
you may want to grant yourself access to this bucket? just add your user ARN. probably something like, `arn:aws:iam:: 563295687221:user/eva.monson@sagebase.org`

Reply:
actually that's not necessary since your account already has access to all buckets in sandbox